app/controllers/api/functions.php

```diff
@@ -965,6 +966,12 @@
             ->setParam('functionStatus', $execution->getAttribute('status', ''))
             ->setParam('functionExecutionTime', $execution->getAttribute('time') * 1000); // ms
 
+        $roles = Authorization::getRoles();
+        $isPrivilegedUser = Auth::isPrivilegedUser($roles);
+        $isAppUser = Auth::isAppUser($roles);
+        $execution->setAttribute('stdout', ($isPrivilegedUser || $isAppUser) ? $executionResponse['stdout'] : '');
+        $execution->setAttribute('stderr', ($isPrivilegedUser || $isAppUser) ? $executionResponse['stderr'] : '');
```

Comment:
Let's be consistent with the way we set these attributes.

In the listExecutions endpoint, we use an if condition. Let's stick to the same here as well for consistency